src/ngMock/angular-mocks.js

```diff
@@ -1381,15 +1381,22 @@ function createHttpBackendMock($rootScope, $timeout, $delegate, $browser) {
       function handleResponse() {
         var response = wrapped.response(method, url, data, headers, wrapped.params(url));
         xhr.$$respHeaders = response[2];
-        callback(copy(response[0]), copy(response[1]), xhr.getAllResponseHeaders(),
```

Comment:
I wonder why we are coyping stuff that theoretically are primitive (such as `status`, `statusText`).